.circleci/config.yml

```diff
@@ -27,45 +26,113 @@ jobs:
             - ./packages/shared/node_modules
             - ./packages/webapp/node_modules
             - ./packages/extension/node_modules
+  build:
+    docker:
+      - image: cimg/node:18.16.0
+    resource_class: medium+
```

Comment:
Did you try with a smaller one now?

Reply:
Actually seeing they fully split, maybe even small can do?

Reply:
I did try to optimize resources but i also changed couple things after that, so I can try again.
Specifically about build it fails with anything less than medium+. But maybe I can try again other flows